Automattic/newspack-plugin
# Fix/ci build

### All Submissions:






### Changes proposed in this Pull Request:



Updates `newspack-script` to a version which handles the weird `HookWebpackError: error:0308010C:digital envelope routines::unsupported` error, [as seen here](https://app.circleci.com/pipelines/github/Automattic/newspack-plugin/5484/workflows/41c1fbe0-f1ff-4bff-9949-97e45aaa10dd/jobs/25721/parallel-runs/0/steps/0-105).

Because it's a bump from `4.0.0` to `4.3.6`, there are some other tweaks as well. 

### How to test the changes in this Pull Request:

The real test will be whether the CI job ran on `master` (after this is merged) build the JS files successfully. In the meantime, here's how it can be approximated:

1. Visit [CI for this branch](https://app.circleci.com/pipelines/github/Automattic/newspack-plugin?branch=fix%2Fci-build)
2. Go to the latest `all` workflow and select `build` step
3. Rerun the step with SSH
4. Ensure the step runs to completion, meaning it installs all `npm` dependencies
5. `cd project` and run `npm run build` – observe the files building without error

### Other information:
